src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -139,10 +139,14 @@ private function tryGetVersionNumberFromComposerJson(&$version) : bool
       *
       * @param string $version The by-reference $version variable to assign the version number to, if found.
       * @return bool true if successful; otherwise false.
+      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
       */
     private function tryGetVersionNumberFromComposerLock(&$version) : bool
     {
         $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        if (!file_exists($composerLockPath)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738)